website/project/metadata/hypothesis-testing-studies-using-youth-data.json

```diff
@@ -58,29 +58,34 @@
 		"example_text": "",
 		"required": false
 	}, {
-		"help_text": "",
+		"help_text": "Enter the datasets planned for this analysis",
 		"block_type": "question-label",
-		"display_text": "Enter the datasets planned for this analysis",
+		"display_text": "Datasets",
 		"example_text": "Provide the same information as it was entered on the data request form, copy and paste is preferred.",
 		"required": false
 	}, {
 		"help_text": "",
 		"block_type": "long-text-input",
 		"display_text": "",
 		"example_text": "",
-		"required": false
+		"required": true
 	}, {
 		"help_text": "",
-		"block_type": "question-label",
+		"block_type": "section-heading",
 		"display_text": "Approved data request file",
-		"example_text": "Upload the data request form output pdf file here",
+		"required": false
+	}, {
+		"help_text": "",
+		"block_type": "paragraph",
+		"display_text": "Upload the data request form output pdf file here",
+		"example_text": "",
```

Comment:
This was worded in a confusing way in the spreadsheet. They just meant that the "Upload the data request form output pdf file here" should be the help text, not the example text -- i.e. it should be "permanently displayed" on the input form instead of needing to click the "see example". No need for a separate paragraph block